test/integration.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
Not a fan of embedding our docker stuff into the scripts. We should be calling docker at the top-level.